### ISSUE TITLE
 Time consuming

### ISSUE THREAD

**Issue by RingZing** [milesial/Pytorch-UNet#432]
Although no error is reported, it runs very slowly, and the round progress bar does not advance during training epochs.

**Comment by milesial** [comment]
Hi,
- what GPU do you have?
- what docker image are you using?
- what docker flags are you using to run the container?
- are you sure the training happens on the GPU and not the CPU?

**Comment by genggeng-123** [comment]
I'm having the same issue too.

**Comment by YeYQ22** [comment]
Yeah,i also have the same question ,I set the training batchsize to 4, a total of about 4500 pieces of training data, according to the speed of training, training rounds take about four hours or so, this speed is unacceptable, I use a graphics card is 2080Ti, the parameters of the configuration as shown in Figure
![image](https://github.com/milesial/Pytorch-UNet/assets/118253577/0965d5d2-da64-47b0-a361-28ae585697ac)

**Comment by lxf-code** [comment]
I'm having the same issue too.

**Comment by mvizosopatino** [comment]
I am having the same issue too
